x-pack/test/security_solution_api_integration/test_suites/investigation/timeline/tests/timeline.ts

```diff
@@ -60,16 +60,16 @@ export default function ({ getService }: FtrProviderContextWithSpaces) {
         ).to.equal(0);
       });
     });
-    describe('resolve timeline', () => {
+    describe.skip('resolve timeline', () => {
```

Comment:
consider creating a ticket for this test

Reply:
done here : 714e22d2820d417536d033dda3ec6758ba8ec590